### ISSUE TITLE
Accessibility issue with TabView control

### ISSUE THREAD

**Issue by mrlacey** [microsoft/microsoft-ui-xaml#2995]
**Describe the bug**

The `TabView` control fails basic accessibility tests because it does not implement [the selection pattern](https://docs.microsoft.com/en-us/windows/win32/winauto/uiauto-implementingselection).

**Steps to reproduce the bug**

1. Add a TabView control to a page

```xml
    <winui:TabView>
        <winui:TabViewItem Header="first"/>
        <winui:TabViewItem Header="second"/>
    </winui:TabView>
```

2. Run app
3. Test the app with Accessibility Insights for Windows

**Expected behavior**

The default behavior of the control should not contain anything that causes it to fail basic accessibility checks.
Specifically [Section 508 502.3.10](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology).

**Screenshots**

![image](https://user-images.githubusercontent.com/189547/88543197-830d5f80-d00f-11ea-89a1-e69f51b1e261.png)

**Version Info**

NuGet package version: Microsoft.UI.Xaml 2.4.2

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (20175)              |  Yes  |
| May 2020 Update (19041)            |  Yes   |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            |  Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

This was discovered while reviewing the accessibility of applications generated by Windows Template Studio. (https://github.com/microsoft/WindowsTemplateStudio/issues/3723)

**Comment by chingucoding** [comment]
This was fixed with #2856 .

**Comment by Felix-Dev** [comment]
@mrlacey This is fixed by PR https://github.com/microsoft/microsoft-ui-xaml/pull/2856 I believe.

**Comment by mrlacey** [comment]
Closing as duplicate of #2856 but still happens with 2.5.0-prerelease.200708003
